src/PrestaShopBundle/Controller/Admin/Sell/Order/OrderController.php

```diff
@@ -1224,21 +1223,29 @@ private function getErrorMessages(Exception $e)
                 $e->getMessage(),
                 'Admin.Orderscustomers.Notification'
             ),
-            InvalidRefundQuantityException::class => [
-                InvalidRefundQuantityException::EMPTY_QUANTITY => $this->trans(
+            InvalidRefundException::class => [
+                InvalidRefundException::INVALID_QUANTITY => $this->trans(
                     'Please enter a positive quantity to proceed with your refund.',
                     'Admin.Orderscustomers.Notification'
                 ),
-                InvalidRefundQuantityException::QUANTITY_TOO_HIGH => $this->trans(
+                InvalidRefundException::QUANTITY_TOO_HIGH => $this->trans(
                     'Please enter a maximum quantity of [1] to proceed with your refund.',
                     'Admin.Orderscustomers.Notification',
                     ['[1]' => $refundableQuantity]
                 ),
+                InvalidRefundException::INVALID_AMOUNT => $this->trans(
```

Comment:
Hello @LouiseBonnard I added these new wordings, let me know if you think they need to be modified Thank you